src/lib/tooltip/tooltip.ts

```diff
@@ -392,6 +427,113 @@ export class MdTooltip implements OnDestroy {
+   * If an element has already been created for this message, increase that registered message's
+   * reference count.
+   */
+  private _registerA11yMessage() {
+    if (!this.message) { return; }
+
+    const a11yMessage = a11yMessages.get(this.message);
+    if (a11yMessage) {
+      a11yMessage.count++;
+    } else {
+      this._createA11yMessageElement(this.message);
+    }
+
+    this._setAriaDescribedBy();
+  }
+
+  /**
+   * Removes the a11y tooltip message element if no other tooltips are registered with the same
+   * message. Otherwise, decrease the reference count of the registered a11y tooltip message.
+   */
+  private _unregisterA11yMessage(message: string) {
+    if (!this._platform.isBrowser || !this.message) { return; }
+
+    const a11yMessageElement = a11yMessages.get(message)!;
+    a11yMessageElement.count--;
+
+    // Remove the a11y message if this was its last unique instance.
+    if (a11yMessageElement.count == 0) {
+      this._deleteA11yMessageElement(message);
+    }
+
+    // If the global messages container no longer has any children, remove it.
+    if (!this._getA11yMessagesContainer()!.childNodes.length) {
+      this._renderer.removeChild(document.body, this._getA11yMessagesContainer());
+    }
+
+  }
+
+  /**
+   * Creates a new element in the visually hidden a11y message container element with the message
+   * as its content.
+   */
+  private _createA11yMessageElement(message: string) {
+    if (!this._platform.isBrowser) { return; }
+
+    // Create a visually hidden container for the accessibility messages if one does not yet exist.
+    if (!this._getA11yMessagesContainer()) {
+      this._createA11yMessagesContainer();
+    }
+
+    const messageElement = this._renderer.createElement('div');
+    messageElement.id = `md-tooltip-message-${latestA11yMessageId++}`;
+    messageElement.textContent = message;
+
+    this._getA11yMessagesContainer()!.appendChild(messageElement);
+    a11yMessages.set(message, {element: messageElement, count: 1});
+
+    return messageElement;
+  }
+
+  /** Deletes the a11y message element from the global a11y tooltip messages container. */
+  private _deleteA11yMessageElement(message: string) {
+    if (!this._platform.isBrowser) { return; }
+
+    const a11yMessage = a11yMessages.get(message)!;
+    this._renderer.removeChild(this._getA11yMessagesContainer()!, a11yMessage.element);
+    a11yMessages.delete(message);
+  }
+
+  /** Returns the trigger's aria-describedby attribute. */
+  private _getAriaDescribedby(): string {
+    return this._elementRef.nativeElement.getAttribute('aria-describedby');
+  }
+
+  /** Sets the trigger's aria-describedby attribute to the tooltip message element. */
+  private _setAriaDescribedBy() {
+    // Return if an aria-describedby already exists and it is not referencing a tooltip message.
+    const ariaDescribedBy = this._elementRef.nativeElement.getAttribute('aria-describedby');
```

Comment:
Something I realized just now is that rather than clobbering an existing aria-describedby, we should augment it with the new ID (the attribute can have multiple, comma-delimited IDs). I talked to @mmalerba about doing the same thing for inputs with `md-error` and `md-hint`. We should create a couple utility functions in cdk/a11y to add/remove an ID to an element's  aria attribute. Something like...
```ts
addAriaReferencedId(element, id, 'aria-describedby');
removeAriaReferencedId(element, id, 'aria-describedby');
```
(open to coming up with a better name)